docs/user-documentation/resource-nodes.md

```diff
@@ -165,4 +168,4 @@ For any node, its **Children** tab can be used to see all its members.  You can
 bulk on members using the checkboxes and the Actions dropdown as well as clicking
 on the **Re-order Children** tab to adjust the order in which they display.
```

Comment:
```suggestion
on the **Reorder Children** tab to adjust the order in which they display.
```
The hyphen in "Re-order" looks weird to me. Shouldn't it be "reorder", should it?